integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala

```diff
@@ -1001,6 +1001,12 @@ class CarbonSqlParser()
       "SERIALIZATION_NULL_FORMAT",
       "ALL_DICTIONARY_PATH"
     )
+    if (options.exists(_._1.equalsIgnoreCase("COLUMNDICT")) &&
+      options.exists(_._1.equalsIgnoreCase("ALL_DICTIONARY_PATH"))) {
+      val errorMessage = "Error: COLUMNDICT and ALL_DICTIONARY_PATH can not be used together" +
+        " in options"
+      throw new MalformedCarbonCommandException(errorMessage)
+    }
```

Comment:
@Jay357089 .....Move this newly added code below if loop for not isSupported check. I think ideally first we should validate the options whether they are among the ones allowed for load command and then perform other checks.